### PR TITLE
Version 2.2

### PR DESCRIPTION
**Pull request by mjpost**
Adds WMT21, exposes metadata via `--echo`, and allows empty references.

### CODE REVIEW

File: sacrebleu/sacrebleu.py [hunk 1]
```diff
@@ -80,8 +80,11 @@ def parse_args():
                             help='Use a subset of sentences whose document annotation matches a given regex (see SUBSETS in the source code).')
     arg_parser.add_argument('--download', type=str, default=None,
                             help='Download a test set and quit.')
-    arg_parser.add_argument('--echo', choices=['src', 'ref', 'both'], type=str, default=None,
-                            help='Output the source (src), reference (ref), or both (both, pasted) to STDOUT and quit.')
+    arg_parser.add_argument('--echo', nargs="*", type=str, default=None,
+                            help='Output the source (src), reference (ref), or other avaliable field (docid, ref:A, ref:1 for example) to STDOUT and quit. '
+                                 'You can get avaliable fields with options `--list` and `-t`' 'For example: `sacrebleu -t wmt21 --list`. '
+                                 'If multiple fields are given, they are outputted with tsv format in the order they are given.'
+                                 'You can also use `--echo all` to output all avaliable fields.')
```
Comment (ozancaglayan) [inline sacrebleu/sacrebleu.py:88]: `avaliable -> available` in three places (line 84, 85, 86)

File: test.sh [hunk 2]
```diff
@@ -273,7 +274,7 @@ if [ -z $SKIP_CHRF ]; then
           # Test chrF
           score=$(cat $txt | ${CMD} -w 4 -t wmt17 -l $source-$target -b --metrics chrf --chrf-eps-smoothing)
           expected_score=${CHRF[$name]}
-          echo "import sys; sys.exit(1 if abs(${score}-${expected_score}) > 1e-6 else 0)" | python
+          echo "import sys; sys.exit(1 if abs(${score}-${expected_score}) > 1e-6 else 0)" | python3
```
Comment (ozancaglayan) [inline test.sh:277]: why is this needed? is there any place still where `python` refers to python 2.x?

**Review by martinpopel — APPROVED** [review]
Looks good. I don't have time for a more detailed review now.

File: sacrebleu/sacrebleu.py (hunk 1 above)
Comment (mjpost) [reply sacrebleu/sacrebleu.py:88]: Fixed.

File: test.sh (hunk 2 above)
Comment (mjpost) [reply test.sh:277]: `python` points to `python2` on Mac OS X. I just added a better general fix.

**Review by ozancaglayan — APPROVED** [review]

**Comment by ozancaglayan** [comment]
@mjpost could you quickly check the issue #181 by the way and see whether what I suggest makes sense?

**Comment by mjpost** [comment]
Ugh. The build action keeps failing on the checksum of an IWSLT file, but it works fine when I run it locally.

**Comment by mjpost** [comment]
Finally!

**Comment by ozancaglayan** [comment]
thanks @mjpost !

**Comment by mjpost** [comment]
Very grateful for your continued involvement!
